falcon/testing/helpers.py

```diff
@@ -499,7 +503,13 @@ def create_scope(path='/', query_string='', method='GET', headers=None,
     if include_server:
         scope['server'] = iter([host, port])
 
-    _add_headers_to_scope(scope, headers, content_length, host, port, scheme, http_version)
+    # NOTE(myusko): Clients discard Set-Cookie header
+    #  in the response to the OPTIONS method.
+    if method == 'OPTIONS' and cookies is not None:
+        cookies = None
```

Comment:
As I understand this is the lone coverage miss.
Are we testing outcomes of this branch?

Reply:
You're right! I didn't cover it, will do it, and after this, coverage will be satisfied, nice catch :+1: